Assets/Scripts/HostSession.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine.UI;

```

Comment:
Is there a callback for if the room already exists? We may want to add an error message for this.